From 3d541e54c10f4f61f82f4703487626bea2f61465 Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@cloudera.com>
Date: Tue, 15 May 2012 14:41:25 -0700
Subject: [PATCH 1152/1357] MR1: CLOUDERA-BUILD. regression: unable to read har created with mr1

  Reason: HAR files created with MR1 tools had wrong version
  Author: Alejandro Abdelnur
  Ref: CDH-5839
---
 .../org/apache/hadoop/tools/HadoopArchives.java    |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/src/tools/org/apache/hadoop/tools/HadoopArchives.java b/src/tools/org/apache/hadoop/tools/HadoopArchives.java
index 6e4d578..eea2e95 100644
--- a/src/tools/org/apache/hadoop/tools/HadoopArchives.java
+++ b/src/tools/org/apache/hadoop/tools/HadoopArchives.java
@@ -641,6 +641,9 @@ public class HadoopArchives implements Tool {
     }
   }
   
+  // Hardcoding HAR version here because HARFileSystem comes from Hadoop2
+  private static final String HAR_VERSION = "1";
+	
   /** the reduce for creating the index and the master index 
    * 
    */
@@ -676,7 +679,7 @@ public class HadoopArchives implements Tool {
         }
         indexStream = fs.create(index);
         outStream = fs.create(masterIndex);
-        String version = HarFileSystem.VERSION + " \n";
+        String version = HAR_VERSION + " \n";
         outStream.write(version.getBytes());
         
       } catch(IOException e) {
-- 
1.7.0.4

